workspaces/spectacle/src/spectacle.ts

```diff
@@ -232,6 +237,51 @@ export async function makeSpectacle(opticContext: IOpticContext) {
         );
         return Promise.resolve(endpointChanges);
       },
+      field: (
+        parent: any,
+        { fieldId }: { fieldId: string },
+        context: GraphQLContext
+      ) => {
+        const fieldNodeWrapper = context
+          .spectacleContext()
+          .shapeQueries.findNodeById(fieldId);
+
+        if (
+          !fieldNodeWrapper ||
+          fieldNodeWrapper.result.type !== shapes.NodeType.Field
+        )
+          return Promise.resolve(null);
```

Comment:
I don't actually think we need Promise.resolves here - but that's how the rest of the resolvers are set up - just an fyi

Reply:
I was just assuming all resolvers are supposed to be async as they all seem to be, but 🤷‍♂️.